spec/fixtures/spec-repos/trunk/CocoaPods-version.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
@igor-makarov shoud this be checked in?

Reply:
Yeah. It's definitely in use.

Reply:
do we need to check it in though? Would it change each time a new version is released and have to be committed?

Reply:
I guess what I am asking is if we should let tests download it each time and add it to `.gitignore`

Reply:
I guess that works too. I dunno, this bit isn't really exact science 😂